packages/nodes-base/nodes/Google/Gmail/Gmail.node.ts

```diff
@@ -0,0 +1,254 @@
+import { IExecuteFunctions } from 'n8n-core';
+import {
+	IDataObject,
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+import {
+	googleApiRequest,
+	encodeEmail,
+} from './GenericFunctions';
+
+import {
+	emailOperations,
+	emailFields
+} from './EmailDescription';
+
+import {
+	labelOperations,
+	labelFields,
+} from './LabelDescription';
+
+import * as _ from 'lodash';
```

Comment:
If you're only using `isEmpty`, perhaps it'd be better to only `import { isEmpty } from 'lodash';`?

Reply:
Haha to be honest I used this code from Ricardo's Mandrill code, and you totally right that I should just use the built-in map. Just pushed a fix!